src/loading.jl

```diff
@@ -26,11 +36,11 @@ If the desired device is CUDA (as determined by `ClimaComms.device()`), try load
 """
 macro import_required_backends()
     return quote
-        @static if $mpi_is_required() && $mpi_ext_is_not_loaded()
+        @static if $mpi_is_required()
```

Comment:
The second `import MPI` in `import MPI; import MPI` is a no-op.

Reply:
Good point. Let's remove the `@info` statement, too.

Frankly, the same is true about `import`ing any module, why should we make an exception to print info here? Also, this will be very noisy for massively parallel MPI jobs, before the logger has been configured. We just haven't seen this issue yet.

Maybe better we can change it to a `@debug`

Reply:
The info was there to tell users what `@import_required_backends` loaded, if anything. I found it useful to have when debugging MPI/CUDA extension (e.g., when things where not found), but I am okay with removing it given that we have almost completed that.

Reply:
Sounds good, I changed it to `@debug`, I think it would be useful in that way, too.